content/docs/12-enterprise-version/00-on-prem-system-requirements.md

```diff
@@ -66,19 +100,56 @@ The following points needs to be taken care while creating the Tags:
 
 </InfoBox>
 
-### Permissions
 
-The following permissions are required for the account used to install the platform:
+## Privileges 
+
+
+The vSphere user account used in the various Palette tasks must have the minimum vSphere privileges required to perform the task. The **Administrator** role provides super-user access to all vSphere objects. For users without the **Administrator** role, one or more custom roles can be created based on the tasks being performed by the user.
 
 <br />
 <br />
 
-### vSphere Object
+<InfoBox>
+If the network is a Distributed Port Group under a vSphere Distributed Switch (VDS),  ReadOnly access to the VDS without “Propagate to children” needs to be provided.
+</InfoBox>
+
+## Privileges Under Root-Level Role
+
+<br />
+
+<WarningBox>
```

Comment:
This does not need to be in a warningbox. We can add it before the table and make it the introductory text.  We have two boxes in a short order so let's reduce where possible. 

![CleanShot 2022-12-12 at 08 24 28](https://user-images.githubusercontent.com/29551334/207084171-983cc21c-6f28-48eb-aa0a-44569c32a0a4.png)